privaterelay/country_utils.py

```diff
@@ -0,0 +1,273 @@
+import logging
```

Comment:
Note: this code is all un-changed code moved into this new module from `privaterelay.utils` to avoid a circular import.